db/seeds.rb

```diff
@@ -11,3 +11,8 @@
 Tag.create(name: 'Diabético(a)')
 Tag.create(name: 'Emagrecer')
 Tag.create(name: 'Hipertrofia')
+Patient.create(first_name: 'otavio', last_name: 'schwanck', birth_date: Time.zone.now,
```

Comment:
o nome não é schwanck, é schwaznegger

Reply:
# Table of Contents

1.  [Introduction](#org7be841d)
    1.  [Como instalar](#org8bfbb91)



<a id="org7be841d"></a>

# Introduction

Aqui está minha intrudção.  [Clique aqui](http://google.com) para ver o link externo.  Vera [aqui](#org8bfbb91) como instalar.


<a id="org8bfbb91"></a>

## Como instalar